scripts/tardis

```diff
@@ -6,15 +6,11 @@ import numpy as np
 
 import logging
 import argparse
-
-
-
```

Comment:
Don't make changes to files or code that are out of scope for your pull request.